Assets/02.Scripts/02-9. System/CSVReader.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+public class CSVReader
+{
+    static string SPLIT_RE = @",(?=(?:[^""]*""[^""]*"")*(?![^""]*""))";
+    static string LINE_SPLIT_RE = @"\r\n|\n\r|\n|\r";
+    static char[] TRIM_CHARS = { '\"' };
```

Comment:
위 3가지 필드에 대해서 static으로 선언하신 이유가 있을까요?
Read 메서드 내에서만 사용되는 것 같아서요.
만약 외부에서 접근할 일이 없다면,
```suggestion
    private const string SPLIT_RE = @",(?=(?:[^""]*""[^""]*"")*(?![^""]*""))";
    private const string LINE_SPLIT_RE = @"\r\n|\n\r|\n|\r";
    private const char[] TRIM_CHARS = { '\"' };
```
위와 같이 고치는 것도 괜찮을 것 같습니다.
또한 위 필드들에 대해서 외부에서의 접근이 필요하여 static을 유지한다고 해도, 접근 제한자는 생략하지 말아주세요~

Reply:
살펴보니까 CSVReader.Read()를 static메서드로 만드려고 해서 그런 것 같습니다. static을 떼고 대신 CSVReader 인스턴스를 생성하는 쪽으로 수정하겠습니다.